.github/ISSUE_TEMPLATE/07-accessibility-issue.yml

```diff
@@ -0,0 +1,42 @@
+name: Report an accessibility issue or suggest an improvement to the Publishing Design Guide
+description: Use this template to report an accessibility issue or suggest an improvement that can be made to the Publishing Design Guide
+title: "[Enter a brief title of the accessibility issue or suggested improvement]"
+labels: ["accessibility"]
+projects: ["alphagov/97"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Steps to report an accessibilty issue or suggestion
+        - Update the title with a brief description of the issue or suggestion
+        - Enter the details below to the best of your ability 
+        - Link to [How to meet WCAG](https://www.w3.org/WAI/WCAG22/quickref/)
+  - type: textarea
```

Comment:
@calumryan -> Should this move below the dropdown and checkbox?

Reply:
ok to leave here